BuildingBlocks/test/BuildingBlocks.Application.Tests/Mediatr/QuotaEnforcerBehaviorTests.cs

```diff
@@ -1,100 +1,109 @@
 ﻿using Enmeshed.BuildingBlocks.Application.Abstractions.Exceptions;
+using Enmeshed.BuildingBlocks.Application.Abstractions.Infrastructure.UserContext;
 using Enmeshed.BuildingBlocks.Application.Attributes;
 using Enmeshed.BuildingBlocks.Application.MediatR;
 using Enmeshed.BuildingBlocks.Domain;
 using FluentAssertions;
 using MediatR;
 using Xunit;
 
+using DomainMetricStatus = Enmeshed.BuildingBlocks.Domain.MetricStatus;
```

Comment:
By folder, do you mean the `TestData` subclass?
Because if so, then we'd have things like `TestData.MetricStatuses.ThatIsExhaustedFor1Day` which does not sound that good.